lib/Jsrt/ChakraCore.h

```diff
@@ -104,6 +104,53 @@ typedef JsErrorCode(CHAKRA_CALLBACK * FetchImportedModuleFromScriptCallBack)(_In
 /// </returns>
 typedef JsErrorCode(CHAKRA_CALLBACK * NotifyModuleReadyCallback)(_In_opt_ JsModuleRecord referencingModule, _In_opt_ JsValueRef exceptionVar);
 
+/// <summary>
+///     A structure containing information about a native function callback.
+/// </summary>
+struct JsNativeFunctionInfo
```

Comment:
You should also `typedef` this struct as `JsNativeFunctionInfo` since clang won't automatically do so for you.

Reply:
Ok, thanks.